consensus/consensus.go

```diff
@@ -280,7 +274,7 @@ func (consensus *Consensus) saveAcceptedBlock(electedBlockHash common.Uint256) e
 
 	log.Infof("Accepted block height: %d, local ledger block height: %d, sync needed.", block.Header.UnsignedHeader.Height, chain.DefaultLedger.Store.GetHeight())
 
-	elc, loaded, err := consensus.loadOrCreateElection(heightToKey(block.Header.UnsignedHeader.Height))
+	elc, loaded, err := consensus.loadOrCreateElection(block.Header.UnsignedHeader.Height)
```

Comment:
Why changed map's key from hash to height?
We should consider about elected in forked consensus, proposals from different blockChain maybe have same height.

Reply:
The key is not changed, heightToKey will be called inside function loadOrCreateElection